sdk/core/Azure.Core/src/Internal/RehydrationOperation.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+
+namespace Azure.Core
+{
+    internal class RehydrationOperation : Operation
+    {
+        private readonly NextLinkOperationImplementation _nextLinkOperation;
+        private readonly OperationInternal _operation;
+
+        public RehydrationOperation(HttpPipeline pipeline, RehydrationToken? rehydrationToken, ClientOptions options = null)
```

Comment:
If you don't accept a null `rehydrationToken`, should the input parameter be nullable?

Reply:
removed nullable